docs/source/overview.md

```diff
@@ -25,19 +25,58 @@ flowchart LR
     classDef green fill:#97F0B4,stroke:#333
 ```
 
-Data sources for pyctyominer-transform are measurement data created from other cell biology image analysis tools.
+Data sources for CytoTable are measurement data created from other cell biology image analysis tools.
 These measurement data are the focus of the data source content which follows.
 
 ### Data Source Locations
 
 ```{eval-rst}
 Data sources may be provided to CytoTable using local filepaths or remote object-storage filepaths (for example, AWS S3, GCP Cloud Storage, Azure Storage).
 We use `cloudpathlib <https://cloudpathlib.drivendata.org/~latest/>`_  under the hood to reference files in a unified way, whether they're local or remote.
+```
+
+#### Data Source Locations in the Cloud
```

Comment:
```suggestion
#### Cloud Processing

How to use CytoTable when your data source is in the Cloud:
```

Optional, but I think having a simple header is best.

Reply:
Thanks for this! I'll work towards simplifying this. I'm going to add specificity here as `Cloud Data Sources` to delineate from processing being interpreted as "execution / runtime" (which is untested but may be possible through Parsl executor specification currently).